actors/miner/src/sector_map.rs

```diff
@@ -106,26 +106,17 @@ impl PartitionSectorMap {
         partition_idx: u64,
         sector_numbers: Vec<u64>,
     ) -> anyhow::Result<()> {
-        self.add(partition_idx, BitField::try_from_bits(sector_numbers)?.into())
+        self.add(partition_idx, &BitField::try_from_bits(sector_numbers)?)
     }
     /// Records the given sector bitfield at the given partition index, merging
     /// it with any existing bitfields if necessary.
-    pub fn add(
-        &mut self,
-        partition_idx: u64,
-        mut sector_numbers: UnvalidatedBitField,
-    ) -> anyhow::Result<()> {
+    pub fn add(&mut self, partition_idx: u64, sector_numbers: &BitField) -> anyhow::Result<()> {
         match self.0.get_mut(&partition_idx) {
             Some(old_sector_numbers) => {
-                let old = old_sector_numbers
-                    .validate_mut()
-                    .map_err(|e| anyhow!("failed to validate sector bitfield: {}", e))?;
-                let new = sector_numbers
-                    .validate()
-                    .map_err(|e| anyhow!("failed to validate new sector bitfield: {}", e))?;
-                *old |= new;
+                *old_sector_numbers |= sector_numbers;
             }
             None => {
+                let sector_numbers = sector_numbers.clone();
```

Comment:
Sorry, I missed this. Any reason to clone here instead of taking by-value in `add`?

Reply:
because maybe better to receive a reference of bitfield instead of a value can help to reduce the size of parameters. How do you think?

Reply:
In the expected case, we'll have one bitfield per partition. So, if we take it by reference, we'll have to clone it. Cloning is significantly more expensive than copying 36 bytes (the size of a `BitField` object in wasm).

Reply:
All right, I will fix it to pass a value directly.